website/content/api-docs/secret/ldap.mdx

```diff
@@ -409,29 +426,6 @@ $ curl \
     http://127.0.0.1:8200/v1/ldap/role/dynamic-role
 ```
 
-### Read dynamic role configuration
```

Comment:
This was moved to be grouped with the other `/ldap/role/:role_name` endpoints since that is more standard practice in the Vault docs.